### ISSUE TITLE
Feedparser Error - unexpectedly nil on reboot of app

### ISSUE THREAD

**Issue by mitchdowney** [podverse/podverse#23]
Adding screenshots in case they are useful later. Issue happened after I subscribed to a bunch of NPR podcasts, then restarted the app.

![image](https://cloud.githubusercontent.com/assets/2262919/10596732/f77a804e-76ac-11e5-9d49-3dfd67e21c4c.png)

![image](https://cloud.githubusercontent.com/assets/2262919/10596735/fd58b562-76ac-11e5-9b19-da4916abffd1.png)

**Comment by mitchdowney** [comment]
I imagine we don't want the FeedParser using that those ! in line 148 of Feedparser.swift, since apparently the qName or path could be nil...

**Comment by mitchdowney** [comment]
I have not been able to reproduce the error. Apparently it was a podcast I tried to subscribe to from way down the NPR search result list.

In any case, we should still look into a way to prevent that line from being able to crash the app.

**Comment by mitchdowney** [comment]
I ran into this error again. This time it happened after I was subscribing to podcasts in the FindSearchTableView and the app crashed. When I rebooted the app this error displayed. Once this error happens once, the app always crashes when you try to start it.

Perhaps the error happened in this case because of failed feedparsing or downloading?

![image](https://cloud.githubusercontent.com/assets/2262919/10807603/5e034586-7daf-11e5-9276-1d43ebf02912.png)
